### ISSUE TITLE
[Feature] Allow theme color selection with hex code or RGB values 

### ISSUE THREAD

**Issue by yskur413** [OlegWock/anori#221]
I believe that having the ability to simply enter a hex value (or set of RGB values) would make matching a theme's colors to its background image much simpler than using the graphical color selector that is available.

**Comment by sjain882** [comment]
Agreed. As a temporary workaround, however, one can export options, unzip, edit the theme files, re-zip, and re-import them.

**Comment by OlegWock** [comment]
Apologies for the long wait, but this has been finally added as part of v1.27.0 which should be available already (or in couple of days, depending on how quickly your browser updates extensions). More improvements to themes coming in next versions
